.github/workflows/update-progress.yml

```
@@ -0,0 +1,47 @@
+name: Update Progress Bars
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)